DocumentedExamples/Zonally_Averaged_Global_Meridional_Overturning_Circulation.ipynb

```diff
@@ -6,9 +6,9 @@
    "source": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.        psiGM = psiGM.sum(psiGM.cf['longitude'].name)
</pre><p><code>psiGM = gsiGM.cf.sum("longitude")</code> is the cleaner way to write this (and similar throughout) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/217/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Great